.buildbot.config.toml

```diff
@@ -1,16 +1,17 @@
 # Config file for continuous integration.
 
 [rust]
-codegen-units = 0           # Use many compilation units.
-debug-assertions = true     # Turn on assertions in rustc.
+codegen-units = 0            # Use many compilation units.
+debug-assertions = true      # Turn on assertions in rustc.
+debug-assertions-std = false # Assertions in libstd cause too much overhead.
 
 [build]
 docs = false
 extended = true
 tools = ["cargo", "rustdoc"]
 
 [llvm]
-assertions = true           # Turn on assertions in LLVM.
+assertions = true            # Turn on assertions in LLVM.
```

Comment:
I think a space has been inserted accidentally?

Reply:
It is required to align all comments to make @vext01 happy.

Reply:
OK, well, I don't want him to be unhappy :p